lib/thrift/protocol/theader.ex

```diff
@@ -0,0 +1,153 @@
+defmodule Thrift.Protocol.THeader do
+  alias Thrift.Protocol.Binary
+  alias Thrift.TApplicationException
+
+  # THeader format:
+  # https://github.com/apache/thrift/blob/master/doc/specs/HeaderFormat.md
```

Comment:
Maybe this should be part of a moduledoc block?

This link is also duplicated below (above `deserialize`).

Reply:
Thanks, done.